.travis.yml

```diff
@@ -0,0 +1,9 @@
+language: java
+script: mvn test
+notifications:
+  # Email notifications are disabled to not annoy anybody.
+  # See http://about.travis-ci.org/docs/user/build-configuration/ to learn more
+  # about configuring notification recipients and more.
+  email: false
```

Comment:
why don't you enable email notifications? Who will be able to receive the notifications, if we enable it??